python/tank/authentication/shotgun_shared/saml2_sso.py

```diff
@@ -88,7 +125,7 @@ def __init__(self, window_title="SSO"):
         # will only limit the visit to the login flow, it is difficult to
         # display the warning only conditionnally. It is much simpler to hide
         # the message on our side.
-        css_style = base64.b64encode('div.browser_not_approved { display: none !important; }')
+        css_style = base64.b64encode("div.browser_not_approved { display: none !important; }")
```

Comment:
I've clarified the comment, giving more details as to what the issue is and the mitigation taken.